src/techui_builder/screen.py

```diff
@@ -1,4 +1,3 @@
-import math
 import warnings
 import xml.etree.ElementTree as ET
```

Comment:
can `ET` be renamed to `etree` for consistency with other files please?
```suggestion
import xml.etree.ElementTree as etree
```